compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/NameCollision.hs

```diff
@@ -191,6 +188,7 @@ isAscendant (ModuleName xs) (ModuleName ys) =
 -- the module's ascendants and descendants for potential collisions.
 checkModule :: MonadGamma m => Module -> m ()
 checkModule mod0 = do
+    -- TODO(NICK) - check for collisions with TypeSynonyms
```

Comment:
Please include the issue number (i.e. #3616) in the todo comments. That makes it easier to find all of them.